src/contribution-guide/contributing.md

```diff
@@ -0,0 +1,24 @@
+# Contribute to Kubewarden docs
+
+_Kubewarden welcomes suggestions for improvement from all contributors, new and experienced!_
+
+You can contribute to Kubewarden documentation by
+
+- Improving existing content
+- Creating new content
+- (_Upcoming_) Helping with translation of the documentation
+
+## Getting started
+
+Everybody is welcome to suggest changes by submitting a pull request
+or reporting a bug/typo with the help of a GitHub issue to the
+[`kubewarden/docs` repository](https://github.com/kubewarden/docs).
+Knowledge of [git](https://git-scm.com/) and [GitHub](https://lab.github.com/) is
+an essential prerequisite to getting started.
+
+To get involved with the documentation:
+- Familiarize yourself with the [documentation repository](https://github.com/kubewarden/docs)
+and the [static site generator](https://rust-lang.github.io/mdBook/).
+- mdBook uses the [CommonMark spec](https://commonmark.org/) of Markdown.
```

Comment:
@divya-mohan0209 this is referencing mdBook, which we don't use anymore. Maybe we can reword the sentence, after all we're still using markdown

Reply:
Thank you, Flavio for catching this! Addressed this in the latest commit!